lib/multiplexer/wrappers.go

```diff
@@ -37,6 +37,7 @@ type Conn struct {
 	protocol  Protocol
 	proxyLine *ProxyLine
 	reader    *bufio.Reader
+	writeSkip int
```

Comment:
Do you think just skipping amount of bytes without controlling those bytes match prefix is enough?
We most probably should be fine, but if something goes wrong there's possibility for some cryptic bugs 😅

Reply:
You're right, I added a check for that.